freqtrade/tests/optimize/test_backtest_detail.py

```diff
@@ -0,0 +1,181 @@
+# pragma pylint: disable=missing-docstring, W0212, line-too-long, C0103, unused-argument
+import logging
+from unittest.mock import MagicMock
+
+from pandas import DataFrame
+import pytest
+
+
+from freqtrade.optimize.backtesting import Backtesting
+from freqtrade.strategy.interface import SellType
+from freqtrade.tests.optimize import (BTrade, BTContainer, _build_backtest_dataframe,
+                                      _get_frame_time_from_offset)
+from freqtrade.tests.conftest import patch_exchange
+
+
+# Test 0 Minus 8% Close
+# Test with Stop-loss at 1%
+# TC1: Stop-Loss Triggered 1% loss
+tc0 = BTContainer(data=[
```

Comment:
headers here are appreciated ;)

Reply:
i tried to add headers - but it kinda sucks ... not sure if it really gives benefit this way - if you have other proposals sure thing ...

Reply:
adding # O, H, L, C on top of your matrice sucks ?!

Reply:
because it'd be a ton of whitespace - yes
and i don't really see the point to add it for every testcase - maybe once - but ...